src/components/ProofOfAddress/Guidance/style.css

```diff
@@ -2,29 +2,33 @@
 
 .content {
   position: relative;
-  margin: 0 40px 1rem;
+  margin: 0 40*@unit 16*@unit;
   flex: 1;
   flex-direction: column;
   display: flex;
 
   @media (--small-viewport) {
-    margin: 0 24px 1rem;
+    margin: 0 24*@unit 16*@unit;
   }
 }
 
 .subTitle {
-  font-size: 14px;
+  @font-size: 14;
```

Comment:
this logic seems to be used in multiple places. seem like something which is worth abstracting perhaps?

Reply:
I agree. Sadly, I don't think LESS gives us any such capability to abstract :(

Reply:
I see 😢

Reply:
I guess we can still abstract in a crude way, since it's always `@font-size: 14`.

```
@smaller-font-size-unitless: 14

//to export
@smaller-font-size: @@smaller-font-size-unitless*@unit;
@smaller-font-unit: 1em/@smaller-font-size
```

Reply:
Good idea, I've made this change now 🙂 
We now have:

```
@font-size-base
@unit

@font-size-small
@unit-small

@font-size-x-small
@unit-x-small
```